packages/website/sidebars.js

```diff
@@ -14,11 +14,11 @@ module.exports = {
       'plugins',
     ],
     Guides: [
-      'using-query-suggestions-plugin',
-      'using-recent-searches-plugin',
-      'using-algolia-insights-plugin',
-      'creating-multi-source-autocompletes',
-      'using-dynamic-sources-based-on-query',
+      'adding-suggested-searches',
+      'adding-recent-searches',
+      'combining-multiple-result-types',
```

Comment:
We should try to avoid to word "combining" here because we'll likely introduce an API called Combine.

Reply:
https://github.com/algolia/autocomplete.js/pull/430/commits/3584a91502d0743c93f0865d23a8287471806089